x/gamm/pool-models/stableswap/amm.go

```diff
@@ -317,7 +315,7 @@ func solveCFMMBinarySearchMulti(xReserve, yReserve, wSumSquares, yIn osmomath.Bi
 	} else if yIn.Abs().GTE(yReserve) {
 		panic("cannot input more than pool reserves")
 	}
-	// fmt.Printf("solve cfmm xreserve %v, yreserve %v, w %v, yin %v\n", xReserve, yReserve, wSumSquares, yIn)
```

Comment:
do you think its bad form to leave this println's? I find them helpful when going through and debugging, to already have something w/ the info I need available

Reply:
I think that any form of dead code adds some level of complexity. Also, I usually inspect variables with a debugger and add them to watch so the value of prints wasn't obvious to me.

However, not a strong opinion. I can revert if you find it useful.

Reply:
Can we revert?

Reply:
okay done